src/main/kotlin/io/github/inductiveautomation/kindling/utils/TabStrip.kt

```diff
@@ -151,6 +177,32 @@ open class TabStrip : DnDTabbedPane() {
         )
     }
 
+    fun editTabTitle(tabIndex: Int) {
```

Comment:
Minor: Should be private, to ensure external users don't call into it. Especially because we're not enforcing the `isTabClosable` invariant inside this function.